code/game/machinery/computer/camera_advanced.dm

```diff
@@ -84,7 +84,7 @@
 		user.reset_perspective(null)
 		if(eyeobj.visible_icon && user.client)
 			user.client.images -= eyeobj.user_image
-		user.client.view_size.unsupress()
+		user.client?.view_size.unsupress()
```

Comment:
```suggestion
		user.client.view_size.unsupress()
```
With the above review this is not required.

Reply:
> I was led to believe that the client can become null between any two operations, and so it's always a good idea to nullcheck it whenever you are accessing a client field. Is that no longer the case?

Yes that is possible if for example the client disconnects while the remote eye is active. However in our case the duration between client operations is so small (i.e. its all in one method call, no delay/timer between calls) that it won't have time to become null before we are done using it so it should not be an issue.